src/components/Book.js

```diff
@@ -1,26 +1,38 @@
 // Individual Book component
-import React from 'react';
+// import React from 'react';
 import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { removeBook } from '../redux/books/booksSlice';
 
-const Book = ({ title, author, category }) => (
-  <div className="book">
-    <h3>
-      Title:
-      {title}
-    </h3>
-    <p>
-      Author:
-      {author}
-    </p>
-    <p>
-      Category:
-      {category}
-    </p>
-    <button type="button">Delete</button>
-  </div>
-);
+const Book = ({
+  id, title, author, category,
+}) => {
+  const dispatch = useDispatch();
+
+  const handleRemoveClick = () => {
+    dispatch(removeBook(id));
+  };
+  return (
+    <div className="book">
+      <h3>
+        Title:
+        {title}
+      </h3>
+      <p>
+        Author:
+        {author}
+      </p>
+      <p>
+        Category:
+        {category}
+      </p>
+      <button type="button" onClick={handleRemoveClick}>Delete</button>
```

Comment:
- [ ] Please 🙏🏻 adhere to the project requirements, according to which you should add the `onClick` event on the **Button** component, which will help to add or remove the either new book or existing book into the `redux` store using the reducer. Right now you are using the `onClick` event but using `jsx` button tag, not the component, which is fine but not fulfilling the requirement. You should create a Button.js file for creating the component and then import that component and add the `click` event to make things work as it working right now.

### Requirement
![Screenshot from 2023-08-23 21-53-57](https://github.com/MohamedRoshdy2021/Book-store/assets/66476329/c7ad4113-ea5e-4e3f-9794-d95fa9e084dd)